core/src/main/java/org/apache/accumulo/core/file/blockfile/impl/SeekableByteArrayInputStream.java

```diff
@@ -94,13 +96,13 @@ public long skip(long requestedSkip) {
       }
     }
 
-    cur += actualSkip;
+    cur.getAndAdd(actualSkip);
```

Comment:
The skip method is another one where the get and the add are disjointed, and should be atomic.

Reply:
addressed in 67d4fa4